lib/ansible/modules/system/aix_lvg.py

```diff
@@ -0,0 +1,273 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# (c) 2017, Kairo Araujo <kairo@kairo.eti.br>
+# GNU General Public License v3.0+ (see COPYING or
+# https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+DOCUMENTATION = '''
+---
+author: "Kairo Araujo (@kairoaraujo)"
+module: aix_lvg
+short_description: Configure LVM volume groups for AIX
+description:
+  - This module creates, removes or resize volume groups on AIX LVM.
+version_added: 2.5
+options:
+  vg:
+    description:
+    - Volume Group name
+    required: true
+  pvs:
+    description:
+    - List of comma-separated devices to use as physical devices in this volume
+      group. Required when creating or extending ('present' state) the volume
```

Comment:
`present` is a value of a parameter so it should be decorated like this: `C(present)`. The same for the `absent` bellow.